.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ env:
   # Otherwise, set variable to the commit of your branch on
   # opentelemetry-python-contrib which is compatible with these Core repo
   # changes.
-  CONTRIB_REPO_SHA: master
```

Comment:
I presume this needs to be switched back in lockstep with the opentelemetry-contrib changes?

Reply:
Don't think so. The builds will keep passing until a new change that affects both repos needs to be made.